app/assets/javascripts/discourse/app/widgets/quick-access-profile.js

```diff
@@ -80,11 +80,22 @@ createWidgetFrom(QuickAccessPanel, "quick-access-profile", {
       });
     }
 
+    let draftCount;
+
+    if (this.currentUser) {
+      draftCount = this.currentUser.draft_count
```

Comment:
I think we might need a different translation as this won't work for RTL locales.